validator/src/main.rs

```diff
@@ -2119,6 +2125,13 @@ pub fn main() {
                     .default_value("10")
                     .help("Minimum time that the validator should not be leader before restarting")
             )
+            .arg(
```

Comment:
How about adding a `--min-delinquency` argument that defaults to 5% instead?  Great idea overall though!

Reply:
should be max though right? Even the current min_delinquency_percentage should probably be max, because this is the maximum delinquency to accept?